x/mint/module.go

```diff
@@ -162,34 +158,5 @@ func (AppModule) EndBlock(_ sdk.Context, _ abci.RequestEndBlock) []abci.Validato
 	return []abci.ValidatorUpdate{}
 }
 
-// ___________________________________________________________________________
-
-// AppModuleSimulation functions
-
-// GenerateGenesisState creates a randomized GenState of the mint module.
-func (AppModule) GenerateGenesisState(simState *module.SimulationState) {
-	simulation.RandomizedGenState(simState)
-}
-
-// ProposalContents doesn't return any content functions for governance proposals.
-func (AppModule) ProposalContents(simState module.SimulationState) []simtypes.WeightedProposalContent {
-	return nil
-}
-
-// RandomizedParams creates randomized mint param changes for the simulator.
-func (AppModule) RandomizedParams(r *rand.Rand) []simtypes.ParamChange {
-	return nil
-}
-
-// RegisterStoreDecoder registers a decoder for mint module's types.
-func (am AppModule) RegisterStoreDecoder(sdr sdk.StoreDecoderRegistry) {
-	sdr[types.StoreKey] = simulation.NewDecodeStore(am.cdc)
-}
-
-// WeightedOperations doesn't return any mint module operation.
-func (AppModule) WeightedOperations(_ module.SimulationState) []simtypes.WeightedOperation {
-	return nil
-}
```

Comment:
Why are we deleting the simulation? We should keep it

Reply:
It seemed to me that this logic was unused. I'll double-check now and restore if I was wrong

Reply:
randomized gen state is important! Its what means that the simulator will setup with the correct genesis here

Reply:
Restored simulation and made it randomized. Let me know what you think

Reply:
I wouldn't spend much time on this rn personally / using types.DefaultGenesis() is fine for now

I want to come back and rewrite alot of the simulator wiring as part of #1871

Reply:
Addressed all comments by avoiding randomization for every parameter you suggested. Let me know if this is acceptable now